docs/migration/draft-v9-migration-guide.md

```diff
@@ -19,6 +19,11 @@
   In v9, we will streamline this behavior so that passing `undefined` will result in tracing being disabled, the same as not passing the option at all.
   If you are relying on `undefined` being passed in and having tracing enabled because of this, you should update your config to set e.g. `tracesSampleRate: 0` instead, which will also enable tracing in v9.
 
+- **The `autoSessionTracking` option is deprecated.**
+
+  To enable session tracking, it is recommended to unset `autoSessionTracking` and ensure that either, in browser environments the `browserSessionIntegration` is added, or in server environments the `httpIntegration` is added.
+  To disable session tracking, it is recommended unset `autoSessionTracking` and to remove the `browserSessionIntegration` in browser environments, or configure the `httpIntegration` with the `trackIncomingRequestsAsSessions` option set to `false`.
```

Comment:
l: Is this last part about `trackIncomingRequestsAsSessions` specific to server environments?